src/System.Text.Http.Parser/HttpParser.cs

```diff
@@ -95,14 +95,13 @@ public class HttpParser : IHttpParser
 
         private unsafe void ParseRequestLine<T>(T handler, byte* data, int length) where T : IHttpRequestLineHandler
         {
-            int offset;
-            ReadOnlySpan<byte> customMethod;
             // Get Method and set the offset
-            var method = HttpUtilities.GetKnownMethod(data, length, out offset);
-            if (method == Http.Method.Custom)
-            {
-                customMethod = GetUnknownMethod(data, length, out offset);
-            }
+            var method = HttpUtilities.GetKnownMethod(data, length, out var offset);
+
+            ReadOnlySpan<byte> customMethod =
```

Comment:
Was there a particular reason for this change or just code clean up?

Reply:
GetUnknownMethod was taking a local by ref/out and returning a ref-like. That needed to be reversed to make result returnable.  
GetKnownMethod returned ordinary enum so did not need the change. I just made it for consistency with Unknown.